widgets/tabs/styles/default.less

```diff
@@ -42,7 +43,24 @@
 	.sow-tabs-tab-container {
 		background-color: @tabs_container_background_color;
 		padding: @tabs_container_padding;
-		text-align: @tabs_container_tabs_align;
+
+		& when ( @tabs_container_tabs_position = left ), ( @tabs_container_tabs_position = right ) {
+			& when ( @tabs_container_tabs_align = middle ) {
+				display: flex;
+				justify-content: center;
+			    flex-direction: column;
+			}
+			& when ( @tabs_container_tabs_align = bottom ) {
```

Comment:
Quite a bit of duplication here. Only difference between `middle` and `bottom` is the `justify-content` prop.